src/model/index.js

```diff
@@ -88,7 +87,9 @@ export default function(fields) {
       [fields, params] = [RETURNS_ALL_FIELDS, fields];
     }
     return new Query(model, fields, RETURNS_LIST, params);
-  }
+  };
+
+  model.modelName = name;
```

Comment:
"name" is a read-only property of a class so I called it modelName

We could call it "id" or something shorter.